org.eclipse.jdt.debug/model/org/eclipse/jdt/internal/debug/core/breakpoints/JavaMethodBreakpoint.java

```diff
@@ -634,6 +667,13 @@ public void setMarker(IMarker marker) throws CoreException {
 						JDIDebugBreakpointMessages.JavaMethodBreakpoint_0, e));
 			}
 		}
+		boolean checkIfLambda = marker.getAttribute(LAMBDA_BREAKPOINT, false);
```

Comment:
why do we need extra variable `checkIfLambda`, why not assign directly to `isLambdaBreakpoint`?